### PR TITLE
Implement endpoints for current value of human footprint

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/service/basin_subzone.js [hunk 1]
```diff
@@ -149,6 +149,20 @@ module.exports = (basinSubzonePersistence, seService) => {
       return { total_area: subzoneArea[0].area };
     },
 
+    /**
+     * Get the current value of human footprint for the given basin subzone
+     * @param {String} subzoneId basin subzone id
```
Comment (erikasv) [inline src/service/basin_subzone.js:154]: subzoneId is a number

File: src/service/state.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ module.exports = (statePersistence, municipalityService, seService) => {
      * Get information about an strategic ecosystem in an state. Includes:
      * - percentage of the given strategic ecosystem respect the national area
      *
-     * @param {String} stateId environmental authority id
+     * @param {String} stateId state id
```
Comment (erikasv) [inline src/service/state.js:47]: stateId is a number

File: src/service/state.js [hunk 3]
```diff
@@ -155,6 +155,20 @@ module.exports = (statePersistence, municipalityService, seService) => {
       return { total_area: stateArea[0].area };
     },
 
+    /**
+     * Get the current value of human footprint for the given state
+     * @param {String} stateId state id
```
Comment (erikasv) [inline src/service/state.js:160]: stateId is a number

File: src/persistence/protected_area.js [hunk 4]
```diff
@@ -70,6 +70,20 @@ module.exports = (db, { globalBinaryProtectedAreas }) => {
...
 
+    /**
+     * Find the the current value of human footprint in the given protected area category
+     * @param {String} categoryName protected area category
+     *
+     * @returns {Object} Object of current human footprint value.
+     */
+    findCurrentHFValue: async (categoryName, year = 2018) => (
+      db('geo_hf as ghf')
+        .innerJoin('global_binary_protected_areas as gbpa', 'ghf.binary_protected', 'gbpa.binary_protected')
```
Comment (erikasv) [inline src/persistence/protected_area.js:81]: It can't be a simple join for the combined areas

File: src/service/basin_subzone.js (hunk 1 above)
Comment (cazapatamar) [reply src/service/basin_subzone.js]: done

File: src/service/state.js (hunk 2 above)
Comment (cazapatamar) [reply src/service/state.js:47]: done

File: src/service/state.js (hunk 3 above)
Comment (cazapatamar) [reply src/service/state.js]: done

File: src/persistence/protected_area.js (hunk 4 above)
Comment (cazapatamar) [reply src/persistence/protected_area.js]: done

File: src/service/state.js (hunk 2 above)
Comment (erikasv) [reply src/service/state.js:47]: But it still says "String", please fix it before merging

**Pull request merged**